docs/post-call-survey.md

```diff
@@ -2,7 +2,76 @@
 
 You can configure post-call surveys to collect valuable feedback like quality of services or net promoter score after a call ends.
 
-## Use Microsoft Forms
+### Types of Post-Call survey
+
+- [Create a 1-question poll](#onequestionpoll)
+- [Use Microsoft Forms](#msforms)
+- [Use a Custom Survey](#custom)
+
+## <a id="onequestionpoll">Create a 1-question poll</a>
+
+### Configure using environment variables
+
+```
+VV_POSTCALL_SURVEY_TYPE = "onequestionpoll"
+VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TITLE = "Tell us how we did"
+VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_PROMPT = "How satisfied are you with this virtual appointment's audio and video quality?"
+VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TYPE = "likeOrDislike"
```

Comment:
Let's add here the other two types:

VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TYPE = "likeOrDislike" or "rating" or "text"